index.md

```diff
@@ -45,7 +45,7 @@
 Step by step tutorials.
 
 ### [Umbraco Add ons](Add-ons/index.md)
-Everything you need to know about Forms and Courier, the two premium packages developed and supported by HQ.
+Everything you need to know about Forms and Courier - the two premium packages developed and supported by HQ.
```

Comment:
Here the part of the sentence: 'the two premium packages developed and supported by HQ' is known as a **parenthesis** (a word or phrase, inserted as an explanation or afterthought into a passage which is grammatically complete without it) - see what I did there, grammar fans?

So since the sentence 'could be':

Everything you need to know about Forms and Courier.

the additional text is 'parenthesis' ... and you identify this using 'parenthetical punctuation' - but here's the thing, your main parenthetical punctuation marks are either commas, dashes, or brackets - but it's really down to the style of the writing to which option to choose. Commas 'can' get confused with 'other commas' in a sentence, brackets can be considered 'informal' and dashes stark... so your choice here is:

Everything you need to know about Forms and Courier - the two premium packages developed and supported by HQ.
Everything you need to know about Forms and Courier, the two premium packages developed and supported by HQ.
Everything you need to know about Forms and Courier (the two premium packages developed and supported by HQ).

...personally, I like a dash if the 'Parenthesis' ends the sentence.